Cargo.toml

```diff
@@ -62,6 +62,12 @@ include = [
 all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
 
+[package.metadata.cargo_check_external_types]
```

Comment:
Ah, cool, I didn't realise this configuration could go in the Cargo.toml. I'll do that in rustls too.

Reply:
Here's a rustls update: https://github.com/rustls/rustls/pull/1546